src/pycell.rs

```diff
@@ -152,19 +198,18 @@ impl<T: PyClass> PyCellInner<T> {
 /// # let counter = PyCell::new(py, Counter::default()).unwrap();
 /// # pyo3::py_run!(py, counter, "assert counter.increment('cat') == 1");
 /// ```
-#[repr(C)]
-pub struct PyCell<T: PyClass> {
-    inner: PyCellInner<T>,
-    dict: T::Dict,
-    weakref: T::WeakRef,
-}
+#[repr(transparent)]
+pub struct PyCell<'py, T: PyClass<'py>>(PyAny<'py>, PhantomData<RefCell<T>>);
```

Comment:
Wow...

Reply:
Yeah it's to make PyCell also be able to deref to PyAny and to have the same ownership model.

We can probably undo this if you think it's a bad idea :smile:

Reply:
Yeah, it's reasonable. I was just kind of surprised how you resolved this.